### PR TITLE
Permissions

### PR DESCRIPTION
**Pull request by mskrip**
Create dump for groups and resolved permissions for existing views

Solves #7

### CODE REVIEW

File: isnts/urls.py [hunk 1]
```diff
@@ -14,5 +14,6 @@
     url(r'^register/$', views.donor_register),
     url(r'^pass_change/$', views.donor_pass_change),
     url(r'^donors/information/$', views.donor_information),
+    url(r'^nopermission/$', views.error404),
```
Comment (mitas1) [inline isnts/urls.py]: Nie je nahodou 404 not found error? Permissions denied je 500 alebo nieco take tusim

Comment (mskrip) [reply isnts/urls.py]: Ano 403 je permission denied

Comment (mskrip) [reply isnts/urls.py]: je to opravene, prosim review

**Review by mitas1 — APPROVED** [review]

**Comment by mitas1** [comment]
Dobra praca vidno, ze aj premyslas nad cim co robis

**Pull request merged**